src/SDK/Client/Client.ts

```diff
@@ -133,23 +133,7 @@ export class Client {
             account: this.account,
         })
 
-        const promises = [];
-        for (let appName in this.apps) {
-            const app = this.apps[appName];
-            const p = this.platform?.contracts.get(app.contractId);
-            // @ts-ignore
-            promises.push(p);
-        }
-        Promise
-            .all(promises)
-            .then((res) => {
-                this.state.isReady = true
-            })
-            .catch((e) => {
-                console.error('SDK apps fetching : failed to init', e);
-                throw e;
-            });
-
+        this.state.isReady = true;
```

Comment:
Do we still need this flag?

Reply:
At this exact moment, we can I think just rely on this.state.isAccountReady for our isReady method, and therefore indeed ditch this.state.isReady :)

Reply:
Let's do that?